### PR TITLE
Fixed zip code wrapping, static width and height of pet images

### PR DESCRIPTION
**Pull request by punndcoder28**
Fix issue #51  and #50

### CODE REVIEW

**Comment by punndcoder28** [comment]
@redxzeta Sorry i must have missed this. Thanks for pointing it out! I could try giving width as `"90%"` of the card, i tried to do it with `@media` in css but i was not reflecting. Will setting it to `"90%"` be fine?

**Comment by redxzeta** [comment]
yea lets try it out

**Comment by punndcoder28** [comment]
@redxzeta I have updated width to be `"90%"`. I also took the liberty in making the home page cards look the same as in pet section. I hope that is fine.

File: src/components/pets/PetType.js [hunk 1]
```diff
@@ -212,6 +220,12 @@ export default function PetType({ token }) {
                       src={img}
                       onMouseEnter={onHoverPhoto}
                       onMouseLeave={onBlurPhoto}
+                      style={{
```
Comment (redxzeta) [inline src/components/pets/PetType.js:223]: can you put this part in css in `pets.css`?
you could put it like
 ```
 .petList__container .card-img-top {
  //css here
}
```

File: src/components/home/Home.js [hunk 2]
```diff
@@ -17,17 +17,36 @@ export default function Home({ token }) {
...
-          <Card>
-            <Card.Img variant="top" alt={link ? pet.type : `${pet.type} placeholder`} src={link ? link.medium : placeholder} />
-            <Card.Body>
-              <Card.Title>{pet.name}</Card.Title>
+          <Card style={{ margin: 10, paddingTop: 10, width: "100%" }}>
+            <Card.Img
+              variant="top"
+              alt={link ? pet.type : `${pet.type} placeholder`}
+              src={link ? link.medium : placeholder}
+              style={{
```
Comment (redxzeta) [inline src/components/home/Home.js:25]: put this part in `home.css`
```
.home__container .card-img-top {
//css here

}
```

**Review by redxzeta — CHANGES_REQUESTED** [review]
Great changes. Just need to add the style to css

File: src/components/home/Home.js (hunk 2 above)
Comment (punndcoder28) [reply src/components/home/Home.js:25]: Tried adding style here but it doesn't reflect on the application.

File: src/components/pets/PetType.js (hunk 1 above)
Comment (punndcoder28) [reply src/components/pets/PetType.js:223]: Should be changed in the next commit

**Comment by punndcoder28** [comment]
@redxzeta I've made the requested changes

**Review by redxzeta — APPROVED** [review]

**Comment by redxzeta** [comment]
alright looks good :)

**Pull request merged**